tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -74,15 +114,60 @@ struct PackagedVar2 : db::SimpleTag {
   using type = tnsr::I<DataVector, Dim, Frame::Inertial>;
 };
 
-template <size_t Dim, SystemType system_type>
+template <size_t Dim>
+struct SimpleNormalizedFaceNormal
+    : db::ComputeTag,
+      ::Tags::Normalized<domain::Tags::UnnormalizedFaceNormal<Dim>> {
+  using base = ::Tags::Normalized<domain::Tags::UnnormalizedFaceNormal<Dim>>;
+  using return_type = typename base::type;
+  static void function(const gsl::not_null<return_type*> result,
+                       const Mesh<Dim - 1>& face_mesh,
+                       const Direction<Dim>& direction) noexcept {
+    for (size_t i = 0; i < Dim; ++i) {
+      result->get(i) =
+          DataVector{face_mesh.number_of_grid_points(),
+                     i == direction.dimension() ? 1.0 * i + 0.25 : 0.0};
```

Comment:
I don't see how this is a normalized face normal... why aren't the components either 1 or 0 ?

Reply:
I'll admit... I didn't bother normalizing it because it technically doesn't matter in this case (just needs to be consistently the same/reversed normal vector). I'm happy to either:
- add a comment
- rename the class
- just normalize the darn thing
I think the last is the least hacky. Thoughts?

Reply:
I'd favor either renaming the class or normalizing the darn thing. Your pick :)